templates/taxi/car_detail.html

```diff
@@ -3,20 +3,31 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user not in car.drivers.all %}
```

Comment:
The condition `user not in car.drivers.all` may not work as expected because `user` is a `User` object and `car.drivers.all()` returns a queryset of `Driver` objects. Ensure that `user` is compared to the appropriate field in `Driver` objects, such as `driver.user` if such a relationship exists.